### ISSUE TITLE
Typo in enterprise.xhtml example: adabapi != adbapi

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11202]
|[<img alt="jyknight's avatar" src="https://avatars.githubusercontent.com/u/873903?s=50" width="50" height="50">](https://github.com/jyknight)| @jyknight reported|
|-|-|
|Trac ID|trac#1670|
|Type|defect|
|Created|2006-04-27 02:47:35Z|

<details><summary>Searchable metadata</summary>

```
trac-id__1670 1670
type__defect defect
reporter__jknight jknight
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1146106055000000 1146106055000000
changetime__1146106362000000 1146106362000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="foom's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|foom set status to `closed`|
|-|-|

(In [16670]) Fix typo reported by Ajeeb.
Closes [#11202](https://github.com/twisted/twisted/issues/11202)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
